### PR TITLE
added user module, task entity, many to one rel.

### PR DESCRIPTION
**Pull request by Andrei-Suta**

### CODE REVIEW

File: src/task/task.entity.ts [hunk 1]
```diff
@@ -1,4 +1,25 @@
...
+	public id: number;
+
+	@Column()
+	public title: string;
+
+	@Column()
+	public description: string;
+
+	@Column()
+	public isDeleted: boolean;
```
Comment (danielursu) [inline src/task/task.entity.ts:18]: @mgandore edited this file

File: package.json [hunk 2]
```diff
@@ -37,6 +38,7 @@
     "@nestjs/cli": "^9.0.0",
     "@nestjs/schematics": "^9.0.0",
     "@nestjs/testing": "^9.0.0",
+    "@types/babel__core": "^7.1.19",
```
Comment (danielursu) [inline package.json:41]: do you need this?

File: src/user/user.enity.ts [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import { Exclude } from "class-transformer";
+import { Column, Entity, OneToMany, PrimaryGeneratedColumn } from "typeorm";
+
+import { TaskEntity } from "src/task/task.entity";
+
+@Entity()
+export class UserEntity {
```
Comment (danielursu) [inline src/user/user.enity.ts]:
```suggestion
export class User {
```

File: src/user/user.enity.ts [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+	@PrimaryGeneratedColumn()
+	public id!: number;
+
+	@Column({ type: "varchar", default: "" })
```
Comment (danielursu) [inline src/user/user.enity.ts]:
```suggestion
	@Column({ type: "text", nullable: false })
```

File: src/user/user.enity.ts [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+	public email: string;
+
+	@Exclude()
+	@Column({ type: "varchar", default: "" })
+	public password: string;
```
Comment (danielursu) [inline src/user/user.enity.ts]:
```suggestion
	@Column({ type: "text", nullable: false })
	public password: string;
```

File: package.json [hunk 6]
```diff
@@ -26,6 +26,7 @@
     "@nestjs/core": "^9.0.0",
```
Comment (danielursu) [inline package.json:26]: .DS_Store to .gitignore

**Review by danielursu — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (Andrei-Suta) [reply package.json:41]: aveam erori legate de babel_core, am instalat pachetul ca sa scap de erori

**Review by mgandore — APPROVED** [review]

**Review by danielursu — APPROVED** [review]

**Pull request merged**